src/app/window.cpp

```diff
@@ -94,6 +100,11 @@ QTabWidget *MainWindow::tabs_widget()
     media->fill_tabs();
     settings->fill_tabs();
 
+    CameraSettingsSubTab *cameraSettings = settings->findChild<CameraSettingsSubTab*>();
```

Comment:
none of this should be in the main window

if you look at how i handle other "cross tab signals", i really think the signal should come from the config to allow any member to "subscribe" to it

Reply:
Are you referring to use of the Config object to broker all of the settings-related signals? It seems logical to me to have the object that owns to the two objects that are communicating with each other to broker communications between them. It feels weird to have a static singleton object managing instances of other classes but I'll have to think more about that.

Reply:
I guess I half see your point. It is good to have a place where you can go to subscribe to whatever settings you care about. I am having trouble reconciling the singleton-versus-non-singleton dichotomy.

Reply:
yeah, thats exactly the way i see it

its not really about the config object (which in the future i would want all this to be handled by some sort of "manager" or "agent") managing the other classes, more of a "master class" triggering various events